tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRTransforms.java

```diff
@@ -131,6 +131,11 @@ class DependencyOrder {
 
         Relation<Matcher, String> dependsOn = Relation.of(new LinkedHashMap<Matcher, Set<String>>(), LinkedHashSet.class);
         {
+	    addDependency( "und-Ethi-t-und-latn-m0-alaloc-geminate", "am-Ethi-t-am-ethi-m0-geminate", "und-Ethi-t-und-latn-m0-alaloc" );
+	    addDependency( "und-Ethi-t-und-latn-m0-aethiopi-geminate", "am-Ethi-t-am-ethi-m0-geminate", "und-Ethi-t-und-latn-m0-aethiopi" );
+	    addDependency( "und-Ethi-t-und-latn-m0-beta_metsehaf-geminate", "am-Ethi-t-am-ethi-m0-geminate", "und-Ethi-t-und-latn-m0-beta_metsehaf" );
```

Comment:
why is there an underscore (_) in "beta_metsehaf" here but not elsewhere?

the files und-Latn-t-und-ethi-m0-beta-metsehaf-geminate.txt and und-Ethi-t-und-latn-m0-beta-metsehaf-geminate.xml both have hyphen instead

Reply:
though und-Ethi-t-und-latn-m0-beta_metsehaf.xml has underscore

Reply:
@pedberg-icu and @dyacob my question is maybe outdated by subsequent commits

I defer to Peter about the file names -- or if you'd like me to run tests on this or investigate, just let me know